backend/python/app/models/comment.py

```diff
@@ -0,0 +1,37 @@
+from sqlalchemy import inspect
+from sqlalchemy.orm.properties import ColumnProperty
+
+from . import db
+
+
+class Comment(db.Model):
+
+    __tablename__ = "comments"
+
+    id = db.Column(db.Integer, primary_key=True, nullable=False)
+    story_translation_content_id = db.Column(
+        db.Integer,
+        db.ForeignKey("story_translation_contents.id"),
+        index=True,
+        nullable=False,
+    )
+
+    user_id = db.Column(db.Integer, db.ForeignKey("users.id"), nullable=False)
+
+    comment_index = db.Column(db.Integer, nullable=False)
+    time = db.Column(db.Integer, nullable=False)
+    resolved = db.Column(db.Boolean, nullable=False)
+    content = db.Column(db.String, nullable=False)
```

Comment:
`db.String` doesnt work in mysql, I think you need LONGTEXT from the mysql dialect: https://github.com/uwblueprint/planet-read/blob/main/backend/python/app/models/story_translation_content.py#L15
<details><summary>Error on database schema create:</summary>
<p>
py-backend_1  | 2021-07-12 03:56:28,791 INFO sqlalchemy.engine.base.Engine ()
py-backend_1  | 2021-07-12 03:56:28,826 INFO sqlalchemy.engine.base.Engine COMMIT
py-backend_1  | 2021-07-12 03:56:28,827 INFO sqlalchemy.engine.base.Engine CREATE INDEX ix_story_translation_contents_story_translation_id ON story_translation_contents (story_translation_id)
py-backend_1  | 2021-07-12 03:56:28,827 INFO sqlalchemy.engine.base.Engine ()
py-backend_1  | 2021-07-12 03:56:28,847 INFO sqlalchemy.engine.base.Engine COMMIT
py-backend_1  | Traceback (most recent call last):
py-backend_1  |   File "/usr/local/lib/python3.7/site-packages/sqlalchemy/sql/compiler.py", line 2942, in visit_create_table
py-backend_1  |     create_column, first_pk=column.primary_key and not first_pk
py-backend_1  |   File "/usr/local/lib/python3.7/site-packages/sqlalchemy/sql/compiler.py", line 352, in process
py-backend_1  |     return obj._compiler_dispatch(self, **kwargs)
py-backend_1  |   File "/usr/local/lib/python3.7/site-packages/sqlalchemy/sql/visitors.py", line 96, in _compiler_dispatch
py-backend_1  |     return meth(self, **kw)
py-backend_1  |   File "/usr/local/lib/python3.7/site-packages/sqlalchemy/sql/compiler.py", line 2975, in visit_create_column
py-backend_1  |     text = self.get_column_specification(column, first_pk=first_pk)
py-backend_1  |   File "/usr/local/lib/python3.7/site-packages/sqlalchemy/dialects/mysql/base.py", line 1721, in get_column_specification
py-backend_1  |     column.type, type_expression=column
py-backend_1  |   File "/usr/local/lib/python3.7/site-packages/sqlalchemy/sql/compiler.py", line 402, in process
py-backend_1  |     return type_._compiler_dispatch(self, **kw)
py-backend_1  |   File "/usr/local/lib/python3.7/site-packages/sqlalchemy/sql/visitors.py", line 96, in _compiler_dispatch
py-backend_1  |     return meth(self, **kw)
py-backend_1  |   File "/usr/local/lib/python3.7/site-packages/sqlalchemy/sql/compiler.py", line 3463, in visit_string
py-backend_1  |     return self.visit_VARCHAR(type_, **kw)
py-backend_1  |   File "/usr/local/lib/python3.7/site-packages/sqlalchemy/dialects/mysql/base.py", line 2208, in visit_VARCHAR
py-backend_1  |     "VARCHAR requires a length on dialect %s" % self.dialect.name
py-backend_1  | sqlalchemy.exc.CompileError: VARCHAR requires a length on dialect mysql
py-backend_1  |
py-backend_1  | The above exception was the direct cause of the following exception:
py-backend_1  |
py-backend_1  | Traceback (most recent call last):
py-backend_1  |   File "server.py", line 12, in <module>
py-backend_1  |     app = create_app(config_name)
py-backend_1  |   File "/app/app/__init__.py", line 63, in create_app
py-backend_1  |     models.init_app(app)
py-backend_1  |   File "/app/app/models/__init__.py", line 26, in init_app
py-backend_1  |     db.create_all()
py-backend_1  |   File "/usr/local/lib/python3.7/site-packages/flask_sqlalchemy/__init__.py", line 1039, in create_all
py-backend_1  |     self._execute_for_all_tables(app, bind, 'create_all')
py-backend_1  |   File "/usr/local/lib/python3.7/site-packages/flask_sqlalchemy/__init__.py", line 1031, in _execute_for_all_tables
py-backend_1  |     op(bind=self.get_engine(app, bind), **extra)
py-backend_1  |   File "/usr/local/lib/python3.7/site-packages/sqlalchemy/sql/schema.py", line 4554, in create_all
py-backend_1  |     ddl.SchemaGenerator, self, checkfirst=checkfirst, tables=tables
py-backend_1  |   File "/usr/local/lib/python3.7/site-packages/sqlalchemy/engine/base.py", line 2095, in _run_visitor
py-backend_1  |     conn._run_visitor(visitorcallable, element, **kwargs)
py-backend_1  |   File "/usr/local/lib/python3.7/site-packages/sqlalchemy/engine/base.py", line 1656, in _run_visitor
py-backend_1  |     visitorcallable(self.dialect, self, **kwargs).traverse_single(element)
py-backend_1  |   File "/usr/local/lib/python3.7/site-packages/sqlalchemy/sql/visitors.py", line 145, in traverse_single
py-backend_1  |     return meth(obj, **kw)
py-backend_1  |   File "/usr/local/lib/python3.7/site-packages/sqlalchemy/sql/ddl.py", line 787, in visit_metadata
py-backend_1  |     _is_metadata_operation=True,
py-backend_1  |   File "/usr/local/lib/python3.7/site-packages/sqlalchemy/sql/visitors.py", line 145, in traverse_single
py-backend_1  |     return meth(obj, **kw)
py-backend_1  |   File "/usr/local/lib/python3.7/site-packages/sqlalchemy/sql/ddl.py", line 832, in visit_table
py-backend_1  |     include_foreign_key_constraints,  # noqa
py-backend_1  |   File "/usr/local/lib/python3.7/site-packages/sqlalchemy/engine/base.py", line 1011, in execute
py-backend_1  |     return meth(self, multiparams, params)
py-backend_1  |   File "/usr/local/lib/python3.7/site-packages/sqlalchemy/sql/ddl.py", line 72, in _execute_on_connection
py-backend_1  |     return connection._execute_ddl(self, multiparams, params)
py-backend_1  |   File "/usr/local/lib/python3.7/site-packages/sqlalchemy/engine/base.py", line 1066, in _execute_ddl
py-backend_1  |     else None,
py-backend_1  |   File "<string>", line 1, in <lambda>
py-backend_1  |   File "/usr/local/lib/python3.7/site-packages/sqlalchemy/sql/elements.py", line 481, in compile
py-backend_1  |     return self._compiler(dialect, bind=bind, **kw)
py-backend_1  |   File "/usr/local/lib/python3.7/site-packages/sqlalchemy/sql/ddl.py", line 29, in _compiler
py-backend_1  |     return dialect.ddl_compiler(dialect, self, **kw)
py-backend_1  |   File "/usr/local/lib/python3.7/site-packages/sqlalchemy/sql/compiler.py", line 322, in __init__
py-backend_1  |     self.string = self.process(self.statement, **compile_kwargs)
py-backend_1  |   File "/usr/local/lib/python3.7/site-packages/sqlalchemy/sql/compiler.py", line 352, in process
py-backend_1  |     return obj._compiler_dispatch(self, **kwargs)
py-backend_1  |   File "/usr/local/lib/python3.7/site-packages/sqlalchemy/sql/visitors.py", line 96, in _compiler_dispatch
py-backend_1  |     return meth(self, **kw)
py-backend_1  |   File "/usr/local/lib/python3.7/site-packages/sqlalchemy/sql/compiler.py", line 2956, in visit_create_table
py-backend_1  |     from_=ce,
py-backend_1  |   File "/usr/local/lib/python3.7/site-packages/sqlalchemy/util/compat.py", line 182, in raise_
py-backend_1  |     raise exception
py-backend_1  | sqlalchemy.exc.CompileError: (in table 'comments', column 'content'): VARCHAR requires a length on dialect mysql